app/views/job_applications/index.html.erb

```diff
@@ -14,7 +14,7 @@
 
   <h4>Resume:<%= link_to "Download", rails_blob_path(job_application.resume, disposition: "attachment") %></h4>
 
-  <%= link_to 'View', job_job_application_path(@job, job_application), class: 'button is-small is-link is-outlined' %>
+  <%= link_to 'View', job_job_applications_path(@job, job_application), class: 'button is-small is-link is-outlined' %>
```

Comment:
why does the name have job_job? it doesn't look good, please change it

Reply:
- "job" is the name of one controller.
- "job_application" is name of another controller.
- The route resource for job_application is added under job in routes file, so the path is named as "job_job_application_path".
-